static/js/components/EmploymentForm.js

```diff
@@ -20,8 +20,35 @@ import SelectField from './inputs/SelectField';
 import CountrySelectField from './inputs/CountrySelectField';
 import StateSelectField from './inputs/StateSelectField';
 import type { WorkHistoryEntry } from '../flow/profileTypes';
+import type { Validator, UIValidator } from '../util/validation';
+import type {
+  Profile,
+  BoundSaveProfile,
+  ValidationErrors,
+  BoundUpdateProfile,
+} from '../flow/profileTypes';
+import type { UIState } from '../reducers/ui';
+import type { SetWorkHistoryEdit } from '../containers/ProfileFormContainer';
 
 class EmploymentForm extends ProfileFormFields {
+  props: {
+    profile:                          Profile,
+    ui:                               UIState;
+    updateProfile:                    BoundUpdateProfile,
+    saveProfile:                      BoundSaveProfile,
+    clearProfileEdit:                 () => void,
+    errors:                           ValidationErrors,
+    setWorkDialogVisibility:          () => void,
+    setWorkDialogIndex:               () => void,
+    setWorkDegreeLevel:               () => void,
+    setWorkDegreeInclusions:          () => void,
+    setWorkHistoryEdit:               SetWorkHistoryEdit,
+    deletionIndex:                    number,
+    setShowWorkDeleteAllDialog:       (bool: boolean) => void,
```

Comment:
is there a way that we can avoid specifying the actual name of the argument in these type declarations? this seems like it would add extra pain to renaming arguments with no apparent benefit